lib/Support/Windows/Program.inc

```diff
@@ -70,14 +70,17 @@ ErrorOr<std::string> sys::findProgramByName(StringRef Name,
   SmallVector<wchar_t, MAX_PATH> U16Result;
   DWORD Len = MAX_PATH;
   for (StringRef Ext : PathExts) {
-    SmallVector<wchar_t, MAX_PATH> U16Ext;
-    if (std::error_code EC = windows::UTF8ToUTF16(Ext, U16Ext))
-      return EC;
-
     do {
-      U16Result.reserve(Len);
-      Len = ::SearchPathW(Path, c_str(U16Name),
-                          U16Ext.empty() ? nullptr : c_str(U16Ext),
+      U16Result.resize(Len);
```

Comment:
I see you changed `reserve` to `resize`, though `set_size` is used later to set it to size `Len`.  This wasn't part of the original change in upstream.  Was there a specific reason for this change?

Reply:
changed back to reserve.